apps/api/src/app/gene/gene.service.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable, OnModuleInit} from '@nestjs/common';
+import {PrismaService} from "../prisma.service";
+
+@Injectable()
+export class GeneService {
+  constructor(private prismaService: PrismaService) {}
+
+  async fetchGeneSymbols() {
+    try {
+      const hugoJSON = await this.prismaService.gene.findMany();
+      const symbols = hugoJSON.map((entry) => entry.hugoSymbol);
+      return symbols;
```

Comment:
It's a preference thing usually, but you can just do
```suggestion
      return hugoJSON.map((entry) => entry.hugoSymbol);
```